pkg/network/http/ebpf_main.go

```diff
@@ -121,6 +139,8 @@ func newEBPFProgram(c *config.Config, offsets []manager.ConstantEditor, sockFD *
 }
 
 func (e *ebpfProgram) Init() error {
+	defer e.bytecode.Close()
```

Comment:
@p-lambert Do we need to keep this around for HTTPS inspection of new processes?

Reply:
we don't because we clone an existing program when we need to